### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kwacky1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 **Integrations**
 
-Now on Jira Cloud: https://kwacky1.atlassian.net
+Now on Jira Cloud: https://kwacky1.atlassian.net - Oh No, my trial has expired.
```
Comment (kwacky1) [inline README.md]: Did you know that this has expired?

Comment (kwacky2) [reply README.md]: Really, that sucks!

File: README.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 
-Now with CircleCI Integration
+Now with CircleCI Integration - what does the CircleCI say?
 
 **Jira Tests**
```
Comment (kwacky2) [inline README.md]: Where are the tests?

Comment (kwacky1) [reply README.md]: I don't know

**Pull request closed without merging**